package.json

```diff
@@ -23,6 +23,8 @@
     "ember-ace": "^1.1.0",
     "ember-cli-autoprefixer": "^0.7.0",
     "ember-cli-babel": "^6.3.0",
+    "ember-cli-clipboard": "^0.8.0",
+    "ember-code-snippet": "^1.9.0",
```

Comment:
Let's move `ember-code-snippet` to 1.10.0 - the PR just got merged to support being used as an addon.